src/components/switch/OcSwitch.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <span class="oc-switch">
     <button
+      data-testid="oc-switch-btn"
```

Comment:
We have used `data-test-id` so far in this repo. You could start a discussion (in web discussions, because it applies there as well) if we want to go with `data-testid` instead and accept the fact that `data-test-some-other-prop` would have a different format then.

Reply:
https://github.com/owncloud/web/discussions/5367